server/routes/api.js

```diff
@@ -10,8 +10,13 @@ module.exports = (services) => {
   /* GET stations listing. */
   router.get('/stations', async (req, res) => {
     try {
-      const stations = await services.db.stations.list();
-      res.status(200).send(stations);
+      const stations = (await services.db.stations.list()).map(station => { return {
+        id: station.id,
+        lat: parseFloat(station.latitude),
+        lng: parseFloat(station.longitude),
+      }});
+      console.log(stations);
```

Comment:
@bill-feng Do you want to keep this console log?

Reply:
Yes. I think it's okay for the server to have console log.